linter.sh

```diff
@@ -1,14 +1,23 @@
 #!/usr/bin/env bash
 
-gometalinter ./... 2>&1 | read; [ $? == 1 ]
+if [ -d $GOPATH/src/github.com/alecthomas/gometalinter/  ]; then
+    echo "gometalinter already exists, not downloading."
+else
+    go get -u github.com/alecthomas/gometalinter
+    gometalinter --install
+fi
+
+gometalinter ./... 2>&1 > temp.err
```

Comment:
can we use a meaningful name instead of `temp.err`.  Suggestion: `gometalinter-err.out`